build/grunt.js

```diff
@@ -312,25 +218,19 @@ module.exports = function(grunt) {
         'browserify:watch'
       ]
     },
-    usebanner: {
```

Comment:
I guess we can remove `grunt-banner` as well, though, this made me realize we still keep the vttjs license in the novtt file banner (though we shouldn't bother with it in this PR)

Reply:
https://github.com/videojs/video.js/issues/5526